eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheetSlot.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import {
+  CustomerWithAttendance,
+  SlotInterface,
+  SlotInterval,
+} from "@/types/temp";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+/** @TODO remove this and import same type when it's merged*/
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+
+/** function that calculates start and end time for slot
+ * @param intervals - slot intervals array
+ * @returns overallInterval - { firstStartTime, lastEndTime}
+ */
+const getOverallInterval = (
+  intervals: SlotInterval[]
+): {
+  overallStartTime: string;
+  overallEndTime: string;
+} => {
+  return {
+    overallStartTime: intervals[0].startTime,
+    overallEndTime: intervals[intervals.length - 1].endTime,
+  };
+};
+const AttendanceSheetSlot: React.FC<Props> = ({
+  customers,
+  intervals,
+  type,
+  notes,
+}) => {
+  const classes = useStyles();
+
+  const overallInterval = getOverallInterval(Object.values(intervals));
+
+  return (
+    <Table aria-label="simple table">
+      <TableHead>
+        <TableRow className={classes.root}>
+          <TableCell>
+            {overallInterval.overallStartTime} -{" "}
+            {overallInterval.overallEndTime} {type.toUpperCase()}
+            {notes}
+          </TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {Object.keys(intervals).map((interval) => (
+          <React.Fragment key={interval}>
+            <TableRow>
+              <TableCell component="th" scope="row">
+                {interval}
+              </TableCell>
+            </TableRow>
+            {customers.map(
+              (customer) =>
+                customer.attendedInterval === interval && (
+                  <TableRow key={customer.id}>
+                    <TableCell>
+                      {customer.name} {customer.surname}
+                    </TableCell>
+                  </TableRow>
+                )
+            )}
+          </React.Fragment>
+        ))}
+      </TableBody>
+    </Table>
+  );
+};
+
+// #region Styles//
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "& .MuiTableRow-root": {
```

Comment:
@ikusteu so this doesn't even show up in the inspector

Reply:
I'm not sure what you're trying to achieve here, but here you're targeting all elements with the class `.MuiTableRow-root` nested within the element with the class `root`.The dynamic class name will vary, it's only `root` in js, but you get the point. So in this case (taking into account the jsx code), this way you're targeting an element with class `.MuiTableRow-root` within (nested) your `TableRow` component, which I'm pretty sure doesn't exist (didn't check the generated html). 
What I think you wan here is to target `<TableRow className={classes.root}>`. If that's the case, just lose the nested selector (`& .MuiTableRow-root`) and apply the styles to just the `root` class, in other words
```typescript
root: {
  borderWidth: 20 // in css-in-js, you can write pixels as number instead of string if the value is in px (but `20px` would work as well)
}
```